edgeapplications.yaml

```diff
@@ -2826,48 +2826,54 @@ components:
           previous: previous
         total_pages: 6
         results:
-        - cache_by_cookies: cache_by_cookies
-          device_group:
-          - device_group
-          - device_group
-          enable_query_string_sort: true
-          l2_caching_enabled: true
+        - id: 5
+          name: name
           browser_cache_settings: browser_cache_settings
+          browser_cache_settings_maximum_ttl: 5
           cdn_cache_settings: cdn_cache_settings
+          cdn_cache_settings_maximum_ttl: 2
+          cache_by_query_string: cache_by_query_string
           query_string_fields:
           - query_string_fields
           - query_string_fields
-          adaptive_delivery_action: adaptive_delivery_action
-          name: name
-          enable_caching_for_post: true
+          enable_query_string_sort: true
+          cache_by_cookies: cache_by_cookies
           cookie_names:
           - cookie_names
           - cookie_names
-          id: 5
-          cache_by_query_string: cache_by_query_string
-          browser_cache_settings_maximum_ttl: 5
-          cdn_cache_settings_maximum_ttl: 2
-        - cache_by_cookies: cache_by_cookies
+          adaptive_delivery_action: adaptive_delivery_action
           device_group:
           - device_group
           - device_group
-          enable_query_string_sort: true
+          enable_caching_for_post: true
           l2_caching_enabled: true
+          enable_caching_for_options: true
+          enable_stale_cache: true
+          l2_region: l2_region
+        - id: 5
+          name: name
           browser_cache_settings: browser_cache_settings
+          browser_cache_settings_maximum_ttl: 5
           cdn_cache_settings: cdn_cache_settings
+          cdn_cache_settings_maximum_ttl: 2
+          cache_by_query_string: cache_by_query_string
           query_string_fields:
-          - query_string_fields
-          - query_string_fields
+           - query_string_fields
```

Comment:
We are doing some strange indentation. This is a case where we have just one space, while in other parts we have two or no space. Can we normalize it?